src/che-openshift-authentication.ts

```diff
@@ -86,7 +102,13 @@ export async function start(context: theia.PluginContext) {
     }
 
     function getServerUrl(): Promise<string> {
-        return new Promise<string>((resolve, reject) => {
+        return new Promise<string>(async (resolve, reject) => {
+            if (isHostedChe) {
+                const user = await che.user.getCurrentUser();
+                const osUserResponse = await axiosInstance.get<OsUserResponse>('https://api.openshift.io/api/users?filter[username]=' + user.name);
```

Comment:
I think it's possible to remove declarations of `isHostedChe` and `cheApi` and always ask the API.
```
if (process.env['CHE_API']) {
   const user = await che.user.getCurrentUser();
   const osUserResponse = await axiosInstance.get<OsUserResponse>(process.env['CHE_API'] + '/users?filter[username]=' + user.name);
}
```
WDYT?

Reply:
This request is redundant in case when the plugin doesn't run on Hosted Che